jujugui/static/gui/src/app/app.js

```diff
@@ -2803,6 +2806,7 @@ YUI.add('juju-gui', function(Y) {
         console.log('switching models is not supported in sandbox');
       }
       console.log('switching model connection');
+      this.env.set('available', false);
```

Comment:
@frankban We already have `this.env.get('connected')`; however, that was inadequate as it only signals that the GUI is connected to the model. API calls will still fail because we haven't yet logged in, so we needed a second flag, alongsite `connected`, that would represent that the model is both connected and logged in. Hence `available`.